packages/ember-data/lib/system/relationships/belongs-to.js

```diff
@@ -93,6 +93,14 @@ function belongsTo(modelName, options) {
 
   opts = opts || {};
 
+  if (opts.hasOwnProperty('serialize')) {
+    Ember.warn('You provided a serialize option, this belongs in the serializer. See DS.Serializer and it\'s implementations http://emberjs.com/api/data/classes/DS.Serializer.html');
```

Comment:
Sorry @igorT, I can't figure out how to provide the information you mentioned. I can only get ahold of the ```modelName``` which is the name of the model that the relationship is to. I can't find the name of the model or the property on where the ```belongsTo``` was used.